apps/prairielearn/assets/scripts/instructorAssessmentInstancesClient.ts

```diff
@@ -92,6 +92,19 @@ onDocumentReady(() => {
           $(this).find('.select-time-limit').trigger('change');
         });
     },
+    customSearch: (data: AssessmentInstanceRow[], searchText: string) => {
```

Comment:
What's the reason we can't use https://bootstrap-table.com/docs/api/column-options/#searchformatter instead of implementing a custom search function?

Reply:
I'm good with that! Presumably we'll move to another library before long 🙃